packages/api/src/utils/login.ts

```diff
@@ -60,6 +60,27 @@ export interface AzureAccountProvider {
      * @param subscriptionId user used subscription id
      */
     setSubscription(subscriptionId: string): Promise<void>;
+
+    /**
+     * Get account information
+     */
+    getAccountInfo(): Record<string, string> | undefined;
+
+    /**
+     * Get subscription id
+     */
+    getSubscriptionId(): string | undefined;
```

Comment:
Merge `getSubscriptionId` and `getTenantId` into one? E.g., `getSelectedSubscription`.
Since the tenant id is related to selected subs, and it's convenient if we want to add more fields later, e.g., subs name...

Reply:
Good idea, fixed.